### PR TITLE
cassandra database component working

### PR DESCRIPTION
**Pull request by elisehuard**
based on kixi.hecuba, making sure we can connect to cassandra

### CODE REVIEW

File: src/kixi/heimdall/handler.clj [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
+
+(defn find-user
+  [password-hash username]
+  {:username username :password-hash password-hash})
+
+(defn add-user!
+  []
+  true)
+
+(defn auth-user [ds credentials]
```
Comment (otfrom) [inline src/kixi/heimdall/handler.clj]: Move the domain functions to a separate ns and just have the compojure/ring handling here?

File: src/kixi/heimdall/handler.clj [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
+      unauthed)))
 
 (defroutes app-routes
   (GET "/" [] "Hello World")
   ;; untested
-  #_(GET "/database" [] (let [session (:database app/system)]
-                          (alia/execute session "CREATE KEYSPACE alia
-                       WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 1};")))
+  (GET "/database" [] (let [session (:session (:cassandra-session app/system))]
+                        (alia/execute session "USE heimdall;")))
```
Comment (otfrom) [inline src/kixi/heimdall/handler.clj]: same split suggestion as above

**Comment by otfrom** [comment]
I'm torn a bit between this being super small and splitting the ring/compojure stuff from the rest of the domain functions (and more or less the same with the ones accessing c*). wdyt?

File: src/kixi/heimdall/handler.clj (hunk 1 above)
Comment (elisehuard) [inline src/kixi/heimdall/handler.clj]: definitely, that was me just getting started, was even thinking of bypassing the database altogether for first iteration

**Comment by otfrom** [comment]
For this version I think you can :shipit: but I think we should mark down what our decision is here.

**Comment by elisehuard** [comment]
I haven't made any decisions yet - I feel that we might have to go with several namespaces (complex  enough model), but we can start like that?

**Comment by otfrom** [comment]
@elisehuard that sounds like a good idea to me. For the most part I like to keep bits that talk to frameworks/db/io separate from any transformational code, but sometimes things are small enough that the extra indirection doesn't matter. Sounds like we are agreed though.

:shipit:

**Pull request merged**